<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_Window_UI</class>
 <widget class="QMainWindow" name="Login_Window_UI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QWidget" name="Central_Widget">
   <layout class="QVBoxLayout" name="Central_Widget_Layout">
    <item>
     <widget class="QWidget" name="Main_Panel" native="true">
      <layout class="QVBoxLayout" name="Main_Panel_Layout">
       <item>
        <layout class="QHBoxLayout" name="Close_Button_Layout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="Close_Button">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="All_Res.qrc">
             <normaloff>:/all_images/res/Exit.png</normaloff>:/all_images/res/Exit.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="Login_Panel" native="true">
         <layout class="QHBoxLayout" name="Login_Panel_Layout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Login_Layout">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="Name_Layout">
              <item>
               <widget class="QLabel" name="Name_Label">
                <property name="text">
                 <string>学号</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="Name_LineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="Password_Layout">
              <item>
               <widget class="QLabel" name="Password_Label">
                <property name="text">
                 <string>密码</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="Password_LineEdit">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="Login_Button">
              <property name="text">
               <string>登录</string>
              </property>
              <property name="shortcut">
               <string>Return</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="All_Res.qrc"/>
 </resources>
 <connections/>
</ui>
